README.md

```diff
@@ -85,98 +89,87 @@ The Realm Flutter package name is `realm`
-        ```
-        dart pub add build_runner --dev
-        ```
-    * Enable `realm_generator` by adding a `build.yaml` file to the application.
-        ```yaml
-        targets:
-            $default:
-                builders:
-                    realm_generator|realm_object_builder:
-                        enabled: true
-                        generate_for:
-                            - lib/*.dart 
-        ```
-
-    * Import Realm in a dart file (ex. `catalog.dart`).
-        ```dart
-        import 'package:realm/realm.dart';
-        ```
-
-    * Declare a part file `catalog.g.dart` in the begining of the `catalog.dart` dart file after all imports.
-
-        ```dart
-        import 'dart:io';
-
-        part 'catalog.g.dart'
-        ```
-
-    * Create a data model class
-
-        ```dart
-        @RealmModel()
-        class _Item {
-            @PrimaryKey()
-            late final int id;
-
-            late String name;
-            
-            int price = 42;
-        }
-        ```
-    * Generate RealmObject class `Item` from data model class `_Item`.
-
-        ```
-        dart run realm generate
-        ```
-        A new file `catalog.g.dart` will be created next to the `catalog.dart`.
-        
-        _*This file should be committed to source control_
+* Import Realm in a dart file (ex. `catalog.dart`).
 
-    * Use the RealmObject class `Item` with Realm.
+    ```dart
+    import 'package:realm/realm.dart';
+    ```
 
-        ```dart
-        // Create a Configuration object
-        var config = Configuration(Item.schema);
+* Declare a part file `catalog.g.dart` in the begining of the `catalog.dart` dart file after all imports.
 
-        // Opean a Realm
-        realm = Realm(config);
+    ```dart
+    import 'dart:io';
 
-        // Open a write transaction
-        realm.write(() {
-          var myItem = Item(0, 'Iteam', price: 4);
-          realm.add(myItem);
-          final item = realm.add(Item(1, 'Iteam')..price = 20);
-        });
+    part 'catalog.g.dart'
+    ```
 
-        // Get objects from the realm
+* Create a data model class.
 
-        // Get all objects
-        var items = realm.all<Item>();
-        
-        // Get object by index
-        var item = items[5];
-        
-        // Get object by primary key
-        var itemByKey = realm.find<Item>(0);
+    It should start with an underscore `_Item` and be annotated with `@RealmModel()`
+
+    ```dart
+    @RealmModel()
+    class _Item {
+        @PrimaryKey()
+        late final int id;
+
+        late String name;
         
-        // Filter and sort object
-        var objects = realm.query<Item>("name == 'Special Item'");
-        var mymake = 'Tesla';
-        var objects = realm.query<Item>(r'make == $0', [mymake]);
-        ```
+        int price = 42;
+    }
+    ```
+
+* Generate RealmObject class `Item` from data model class `_Item`.
+
+    ```
+    dart run realm generate
```

Comment:
Should we mention the `--watch` flag here?

Reply:
This should probably be in the docs. this is only a getting started guide.